src/main/resources/messages.properties

```diff
@@ -1,21 +1,27 @@
-typeMismatch.courseConfigurationForm.totalClassMinutes = Must be a valid number.
-Min.courseConfigurationForm.totalClassMinutes = Must be greater than 0.
+typeMismatch.courseConfigurationForm.totalClassMinutes=Must be a valid number.
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge keys "typeMismatch.courseConfigurationForm.defaultMinutesPerSession, typeMismatch.courseConfigurationForm.totalClassMinutes, typeMismatch.makeupForm.entries.minutesMadeUp, typeMismatch.rosterForm.sectionModels.attendances.minutesMissed" that have the same value "Must be a valid number.". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=jproperties%3Aduplicated-values)